### PR TITLE
docs: add a ckb tuning guide

### PR DESCRIPTION
**Pull request by yangby-cryptape**

### CODE REVIEW

**Comment by yangby-cryptape** [comment]
@doitian Please review.

File: website/docs/basics/guides/ckb-tuning-guide.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+During IBD, CKB will fetch all headers before downloading the blocks.
+
+All headers whose blocks haven't been downloaded are saved in the memory at first, but some of them will cache in the disk if the size of headers reaches a limit.
+
+As an example, we can configure the limit by appending the follow configurations into `ckb.toml` to reduce the memory usage:
+
+```toml
+[network.header_map]
+# The maximum size of data in memory (default: 300_000)
+primary_limit = 3000
```
Comment (doitian) [inline website/docs/basics/guides/ckb-tuning-guide.md:23]: Please explain what's the unit here. B or MB?

File: website/docs/basics/guides/ckb-tuning-guide.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+# Disable cache if the size of data in memory less than this threshold (default: 20_000)
+backend_close_threshold = 200
```
Comment (doitian) [inline website/docs/basics/guides/ckb-tuning-guide.md:25]: Please explain what's the unit here. B or MB?

**Review by doitian — CHANGES_REQUESTED** [review]

**Comment by doitian** [comment]
日志可以看下 Grafana 的全家桶 https://github.com/grafana/loki

-- Aligned to O2KR4. Deploy Central Log System To Search Logs Of Testnet Nodes

File: website/docs/basics/guides/ckb-tuning-guide.md (hunk 2 above)
Comment (yangby-cryptape) [reply website/docs/basics/guides/ckb-tuning-guide.md:25]: Count of `Header`.

p.s. I will update the docs later.

Comment (doitian) [reply website/docs/basics/guides/ckb-tuning-guide.md:25]: ping

**Review by doitian — APPROVED** [review]

**Comment by doitian** [comment]
stalled and conflicting

**Pull request closed without merging**